components/navigation/MenuBlocks.tsx

```diff
@@ -0,0 +1,68 @@
+import { useRouter } from 'next/router';
+
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import LinkComponent from '../link';
+import Paragraph from '../typography/Paragraph';
+import Label from './Label';
+
+export interface MenuItem {
+  title: string;
+  href: string;
+  description: string;
+  icon: React.ElementType;
+  className?: string;
+  comingSoon?: boolean;
+  beta?: boolean;
+}
+
+export interface MenuBlocksProps {
+  items?: MenuItem[];
+}
+
+/**
+ * @description Component representing a set of menu blocks.
+ * @param {MenuItem[]} [props.items=[]] - The list of items to be displayed in the menu blocks.
+ */
+export default function MenuBlocks({ items = [] }: MenuBlocksProps) {
+  const router = useRouter();
+
+  return (
+    <>
+      {items.map((item, index) => {
+        const isExternalHref = item.href && item.href.startsWith('http');
+
+        return (
+          <LinkComponent href={item.comingSoon ? '' : item.href} key={index}>
+            <a
```

Comment:
```suggestion
            <span
```

Instead of `a` tag, you can use `span` tag.

Reply:
it is throwing an error beacause span doesnt have target attribute